### PR TITLE
prepare for prod

### PR DESCRIPTION
**Pull request by sebastianbenz**

### CODE REVIEW

**Comment by pbakaus** [comment]
Hot tip, you can use the new draft PR feature :) See https://twitter.com/github/status/1096093162345508864

**Comment by sebastianbenz** [comment]
Oooooh nice! Thanks!

**Comment by sebastianbenz** [comment]
@pbakaus I think this is good to merge - PTAL

There are a few things I plan to do in the next PRs:

* refactor platform.js
* add separate api host for prod

File: package.json [hunk 1]
```diff
@@ -1,9 +1,12 @@
 {
   "name": "amp-dev",
   "version": "1.0.2",
-  "description": "The successor of the former ampproject.org: the all-new amp.dev",
+  "description": "The Amp project website.",
```
Comment (pbakaus) [inline package.json]:
```suggestion
  "description": "The AMP Project website.",
```

**Comment by sebastianbenz** [comment]
Fixed the nit. Thanks for the review.

**Pull request merged**